src/Farmer/Arm/Web.fs

```diff
@@ -270,6 +271,16 @@ type Site =
                            javaContainer = this.JavaContainer |> Option.toObj
                            javaContainerVersion = this.JavaContainerVersion |> Option.toObj
                            phpVersion = this.PhpVersion |> Option.toObj
+                           ipSecurityRestrictions = 
+                                match this.IpSecurityRestrictions with
+                                | [] -> null
+                                | restrictions ->
+                                restrictions
+                                    |> List.mapi (fun index restriction ->
+                                       {| ipAddress = $"%s{restriction.IpAddress.ToString()}/32"
+                                          name = restriction.Name
+                                          action = restriction.Action.ToString()
+                                          priority = index + 1 |}) |> box
```

Comment:
The priorities are going to be set in reverse order to the order the restrictions are added to the builder, due to the use of the `::` operator in the custom operation. Is this behaviour by design? Perhaps it would be worth adding a short sentence in the docs to clarify how priority works?

I assume that you considered the alternative of explicitly having the priority in the type and custom operation. Why did you decide that being implicit, rather than explicit, was better?

Reply:
If we allow IP address ranges (not just /32), then priorities might become more important.

Reply:
It's probably worth updating one of the tests to have multiple IP restrictions, to verify the behaviour.

Reply:
Great points, I think my preferred way to solve this is by adding priority as a property to the IpSecurityRestriction type. 

The order will then be inferred from the priorities, no need to worry about the reverse order any more.
And that would also make setting the priorities exactly available to the users.

And then it makes even more sense to add more test to verify the priorities.